packages/core/src/util/log.ts

```
@@ -21,14 +73,45 @@
+  // Function overloads
+  function logWrapper(obj: Record<string, unknown>): void;
+  function logWrapper(msg: string, ...optionalParams: LogParam[]): void;
+  function logWrapper(
+    msgOrObj: string | Record<string, unknown>,
+    ...optionalParams: LogParam[]
+  ): void {
+    if (typeof msgOrObj === "object") {
+      return structuredLog(prefix)({
+        ...msgOrObj,
+        source: prefix,
+      });
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)